internal/host/host_test.go

```diff
@@ -901,6 +901,10 @@ func masterInventory() string {
 	return masterInventoryWithHostname("master-hostname")
 }
 
+func masterInventoryV6() string {
```

Comment:
Should we verify also `workerInventoryV6`?

Reply:
If we need.  Right now we don't